book/website/communication/social-media/social-media-twitter-tips.md

```diff
@@ -1,7 +1,7 @@
 (cm-social-media-twitter-tips)=
 # Tips for Starting with Twitter
 
-Twitter has 353 million monthly active users.
+Twitter has 450 million monthly active users {cite:ps}`Chabrol2021twitter`.
```

Comment:
This should reference `boa2023twitter` from the bibliography instead of `Chabrol2021twitter`.

Reply:
@BrainonSilicon - I think you have to approve these changes to the repository so that @caroldutra3 can merge :)